test/DormBedEntry.test.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+// Components
+import DormBedEntry from '../client/src/components/DormBedEntry';
+
+it('should render correctly with no props', () => {
```

Comment:
Possibly missing a `describe` block with a callback. 

For example: 
`describe('Test DormBedEntry Component', () => {
  it('should render correctly with no props', () => {
   const component = shallow(<DormBedEntry />);

   expect(component).toMatchSnapshot();
  }
}